### ISSUE TITLE
Keycodes issue for Shortcuts

### ISSUE THREAD

**Issue by acketon** [HackPlan/quark-shell-mac#43]
I'm trying to setup a default shortcut using Command+L and am running into a problem with the keycode. The keycode I need to pass into the addKeyboardShortcut function is 0x25.

In my javascript it works if I hard code it into the function, but I want to be able to store the value so that the user can change it. When I store the keycode as a string: "0x25" and pass the string into the function I get the following error in Xcode: 
[NSTaggedPointerString unsignedIntegerValue]: unrecognized selector sent to instance 0x3532783045

I discovered this because my shortcuts were being stored as JSON data and the 0x25 was being parsed into the integer 37 when I wasn't explicitly making it a string.

Shouldn't addKeyboardShortcut take a javascript string for the keycode? I looked at the code a bit and it looks like it is a NSUInteger but I don't really understand all of that.

**Comment by acketon** [comment]
I think I figured it out. I was trying to store 0x25 in my JSON settings file but I see now that the shortcut recorder returns a decimal integer version of that (37) so if I just store that as an integer and then pass that into the addKeyboardShortcut and Shortcut Recorder pref setup options it properly handles loading the shortcut and setting it up on the next app launch from the saved settings. 

The example in the documentation that shows passing 0x7A (F1 key) was throwing me off thinking I had to pass that into the quark function from javascript and I couldn't figure out how to do that since saving it as a string and passing that string into the quark shortcut function was throwing errors.
